jupyterhub_singleuser_profiles/sizes.py

```diff
@@ -39,10 +42,19 @@ def get_form(self, last_size=None):
                 elif size['resources'].get('limits') is None:
                     size['resources']['limits'] = size['resources']['requests']
 
-                req_mem = size['resources']['requests']['memory']
-                req_cpu = size['resources']['requests']['cpu']
-                lim_mem = size['resources']['limits']['memory']
-                lim_cpu = size['resources']['limits']['cpu']
+                req_mem = size['resources']['requests'].get('memory')
```

Comment:
This whole block can be simplified without all the `if`s - `get()` allows you to set default value if the requested field does not exist (which is what you are doing with the conditions), so it can be made into

```
req_cpu = size['resources']['requests'].get('memory', "Default")
...
```